tests/data/invalid-statements/expected_errors.csv

```diff
@@ -108,3 +108,16 @@ publication_details_publisher_no_publication_date.json,required,$[0].publication
 publication_details_publisher_no_bods_version.json,required,$[0].publicationDetails,bodsVersion
 publication_details_publisher_no_publisher.json,required,$[0].publicationDetails,publisher
 statement_declaration_not_string.json,type,$[0].declaration,declaration
+entity_trust_registeredentity.json,const,$[0].recordDetails.entityType.subtype,subtype
```

Comment:
I thought these and the subsequent lines with 'const' are the ones that are not needed?

Reply:
oops yeah something must have gone wrong with the rebase